org-code-javabuilder/protocol/src/main/java/org/code/protocol/ClientMessageDetailKeys.java

```diff
@@ -0,0 +1,16 @@
+package org.code.protocol;
+
+/** Expected keys in the optional detail object of {@link ClientMessage}s */
+public enum ClientMessageDetailKeys {
```

Comment:
I like the idea of having these in one place, but I wonder if it would be simpler to instead have a class of static strings, since we will only ever use these in their String value, not their enum value. Then we don't need to call `.toString()` every time we use them.